Pages/Shared/_Layout.cshtml

```diff
@@ -89,6 +88,7 @@
         </div>
     </footer>
 
```

Comment:
Added link to bootstrap because some class commands did not work with default in wwwroot folder